### PR TITLE
Bug 1821272: Fix the value of bucket name shown in OBC details page

### PR DESCRIPTION
**Pull request by bipuladh**
Screenshot from 2020-04-14 17-33-27

### CODE REVIEW

File: frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/secret.tsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ export const GetSecret: React.FC<GetSecretProps> = ({ obj }) => {
     const configMap = k8sGet(ConfigMapModel, name, namespace);
     Promise.all([secret, configMap])
       .then((data) => {
+        const bucketName = _.get(data[1], 'data.BUCKET_NAME');
```
Comment (cloudbehl) [inline frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/secret.tsx:24]: use optional chaining.

Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/secret.tsx:24]: Need to backport to 4.3

Comment (cloudbehl) [reply frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/secret.tsx:24]: Will that make sense to do it for a single z release?

Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/secret.tsx:24]: Let's keep it ready for any case.

**Pull request merged**